llvm/lib/Target/NVPTX/NVPTXAsmPrinter.cpp

```diff
@@ -1588,12 +1588,22 @@ void NVPTXAsmPrinter::emitFunctionParamList(const Function *F, raw_ostream &O) {
 
       if (isKernelFunc) {
```

Comment:
Do we want to annotate non-kernel function pointer arguments, too?

Reply:
I can make another MR for non-kernel pointer arguments.

Reply:
SGTM.